public/components/event_analytics/redux/slices/count_distribution_slice.ts

```diff
@@ -26,6 +26,9 @@ export const countDistributionSlice = createSlice({
 
 export const { render } = countDistributionSlice.actions;
 
-export const selectCountDistribution = (state) => state.countDistribution;
+export const selectCountDistribution = createSelector(
+  (state) => state.countDistribution,
+  (countDisState) => countDisState
```

Comment:
what does this second line do, that just takes an argument and returns it

Reply:
Yes as for this selectCountDistribution, I only need the entire countDisState